src/common/components/cards/instance-details.tsx

```diff
@@ -44,7 +46,7 @@ export const InstanceDetails = NamedFC<InstanceDetailsProps>('InstanceDetails',
 
     return (
         <>
-            <table className={reportInstanceTable}>
+            <table className={reportInstanceTable} onClick={() => deps.cardSelectionMessageCreator.toggleCardSelection(result.uid)}>
```

Comment:
Can we use a named function here? Might make this easier to debug in the future.

Reply:
good suggestion. Updated.